README.md

```diff
@@ -89,7 +89,7 @@ the built-in defaults are tailored for the latest version of SUSE Linux Enterpri
 The program will scan, in order, the current working directory, `$HOME/.config`, `/etc` and `/usr/etc` for files named `sap_host_exporter.(yaml|json|toml)`.
 The first match has precedence, and the CLI flags have precedence over the config file.
 
-Please refer to the [example YAML configuration](doc/sap_host_exporter.yaml) for more details.
+Please refer to the [example YAML configuration](default.yaml) for more details.
```

Comment:
Very minor nitpick, but why rename the file from the docs? We can just copy from the original filename.

Reply:
I was thinking we don't copy the doc folder in rpm yes

Reply:
done